ui/src/hooks.js

```diff
@@ -190,7 +190,11 @@ export const useChartSeries = ({
   );
 
   const isLoading = queries.find((query) => query.isLoading);
-  const queriesData = queries.map((query) => query.data);
+  const queriesData = queries
+    .map((query) => {
+      return query.data;
+    })
+    .sort((query1, query2) => (query1.key > query2.key ? 1 : -1));
```

Comment:
To me this sorting is not very straightforward. I am thinking maybe another approach would be restructure the data with the query name as Key?

Reply:
@ChengYanJin You mean using (for example) `ControlPlaneBandwidthOut-${instanceIP}-${startingTimeISO}`?

Reply:
Or instead of putting the queries in an array, to put them in object with the key being the query name? (for example `controlPlaneBandwidthOut`)